test/python/ph/test_graph.py

```
@@ -0,0 +1,212 @@
+    construct_graph,
+    cross_fidelity,
+    determine_routing_fidelitites,
+    generate_beam_splitter_matrix,
+)
+
+
+class TestBarFidelity:
+    """Tests for bar_fidelity."""
+
+    @staticmethod
+    def test_ideal_bs_gives_one():
```

Ruff (0.15.20)

[warning] Missing return type annotation for staticmethod `test_ideal_bs_gives_one`

Add return type annotation: `None`

(ANN205)